ReflectorIOS/Views/ArticleList/ArticleCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ArticleCell.swift
+//  ReflectorIOS
+//
+//  Created by Austin Brown on 9/17/20.
+//  Copyright © 2020 Nikhil Menon. All rights reserved.
+//
+
+import SwiftUI
```

Comment:
I would personally add whitespace between 9 and 10. to differentiate the import and the enum.